Userland/Libraries/LibC/fnmatch.cpp

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) 2021, the SerenityOS developers
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/Format.h>
+#include <fnmatch.h>
+
+int fnmatch(char const*, char const*, int)
+{
+    dbgln("FIXME: Implement fnmatch()");
+    return 0;
+}
```

Comment:
Please configure your editor to add a newline at the end of files (CI will fail due to this)